test/topology_custom/test_view_build_status.py

```diff
@@ -192,7 +192,7 @@ async def test_view_build_status_snapshot(manager: ManagerClient):
 @pytest.mark.asyncio
 async def test_view_build_status_migration_to_v2(request, manager: ManagerClient):
     # First, force the first node to start in legacy mode
-    cfg = {'force_gossip_topology_changes': True}
+    cfg = {'force_gossip_topology_changes': True, 'enable_tablets': False}
```

Comment:
again it's weird that this needs to be specified at all, it should be implicit from `force_gossip_topology_changes`. Maybe we have a bug.

Reply:
https://github.com/scylladb/scylladb/issues/21075